external/ilc-assets/ilc-assets.depproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <NugetRuntimeIdentifier>win10-x64-aot</NugetRuntimeIdentifier>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
+    <FileToInclude Include="System.Diagnostics.Tracing" />
```

Comment:
talked to @ericstj about this. Given that the only two depproj's that deploy to a runtime dir are the runtime.depproj and test-runtime.depproj and we don't package any of them, it's probably best and easier to keep this depproj isolated given that there is no other with the same purpose. We did all agree that the name is wrong, so will change it to be `uapaotredist` instead. Also, this project is just temporary and will go away once we bring those project builds back.